src/clj/ataru/hakija/hakija_routes.clj

```diff
@@ -35,12 +36,27 @@
 
 (defn- fetch-form-by-key [key]
```

Comment:
Hmm, maybe it's time to extract this stuff out of routes? To e.g. hakija-form-service.clj?